src/content/structured/components/cards/code.mdx

```diff
@@ -397,9 +394,9 @@ export const topImage = [
   {
     language: "Web component",
     snippet: `<ic-card 
-  heading="Card heading"
-  subheading="Card subheading" 
-  message="This is an example of a card with a top image."
+  heading="Coffee" 
+  subheading="What is coffee?" 
+  message="Coffee is a brewed drink."
```

Comment:
For the examples with images, could the descriptive text reference that there's an image nearby, like it used to?